### DISCUSSION TITLE
Why was webpki-roots changed to a dependency from a dev-dependency?

### DISCUSSION THREAD

**Discussion by kentSarmiento** [awslabs/aws-sdk-rust#706] [Q&A] [answered]
Hello,

webpki-roots was changed from being a dev-dependency in version 0.52.
https://github.com/awslabs/aws-sdk-rust/commit/c801e96441acf33279843c9d9d57a87413bfb20d#diff-f5a138f880ca4a2c199564b39b36f9e9e2d3951a07699e5fe83024bfc9072167R63

What is the reason for this change?
Is this an oversight? Can webpki-roots be reverted as a dev-dependency?

Just a background, we are using a tool that ensures every dependency we have has a license we can use.
webpki-roots has an MPL-2.0 license which is more restrictive than the licenses we want to use.

In version 0.51, since webpki-roots is only a dev-dependency, this wasn't an issue.

**Comment by rcoh** [comment, marked as answer]
it wasn't by accident, however, it was only to support a doc-test—I'm making a PR to remove it, it will go out in 0.53:
https://github.com/awslabs/smithy-rs/blob/7e666dab8bbec977ff88dd885cd7d50c31a10908/rust-runtime/aws-smithy-client/Cargo.toml#L28-L30

**Reply by rcoh** [reply]
fixed here: https://github.com/awslabs/smithy-rs/pull/2170

**Reply by kentSarmiento** [reply]
Great, thanks for the answer and quick fix!
